kitsune/wiki/templates/wiki/mobile/document.html

```diff
@@ -31,8 +31,13 @@
 {% endif %}
 
 {% block content %}
-  <div class="title-bar">{{ document_title(document) }}</div>
+  {% if breadcrumb_items[1] %}
+    <div class="filter-bar">
+      <a class="return-link" href="{{ breadcrumb_items[1][0] }}">{{ breadcrumb_items[1][1] }}</a>
```

Comment:
I feel like I didn't explain myself very well, let me try with code:

```python
topic = TopicFactory()
subtopic1 = TopicFactory(parent=topic, order=1)
subtopic2 = TopicFactory(parent=topic, order=2)
d = DocumentFactory(topics=[subtopic1, subtopic2])
```

the breadcrumbs would be `[topic.product, topic, subtopic2, d]` in this case. The "dominant topic" mechanism was used to pick subtopic2 over subtopic1. The "back to topic" navigation should go to `subtopic2`. I think that would be `breadcrumb_items[-2]`.